core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -617,10 +617,30 @@ void removeProperty(String tableName, String property)
    *         recently changed properties may not be visible immediately.
    * @throws TableNotFoundException
    *           if the table does not exist
+   * @since 1.6.0
+   * @deprecated since 2.1.0; use {@link #getConfiguration(String)} (String)} instead.
    */
+  @Deprecated(since = "2.1.0")
   Iterable<Entry<String,String>> getProperties(String tableName)
       throws AccumuloException, TableNotFoundException;
```

Comment:
If a default method is used in the interface, most, if not all, of the implementing classes and subclasses can simply remove their implementing method:

```suggestion
   * @since 1.6.0
   */
  default Iterable<Entry<String,String>> getProperties(String tableName)
    throws AccumuloException, TableNotFoundException {
    return getConfiguration(tableName).entrySet();
  }
```

(and the same can be done in NamespaceOperations and its implementing classes and subclasses)